File: Frontend/eShoppingWebUI/src/models/baskets/paymentDetails.ts
export interface PaymentDetails {
    cardNumber: string;
    cardHolderName: string;
    cardExpiration: Date;
    cardSecurityNumber: string;
    cardTypeId: number;
    numberOfInstallments: number;
}

export enum CardType{
    Amex = 1,
    Visa = 2,
    MasterCard = 3
}

File: Backend/src/Services/OrderService/OrderService.Domain/Models/PaymentDetails.cs
namespace OrderService.Domain.Models
{
    public class PaymentDetails
    {
        public string CardNumber { get; set; }
        public string CardHolderName { get; set; }
        public DateTime CardExpiration { get; set; }
        public string CardSecurityNumber { get; set; }
        public int CardTypeId { get; set; }
        public int? NumberOfInstallments { get; set; }
    }
}
